src/scales/scale.logarithmic.js

```diff
@@ -152,6 +135,10 @@ class LogarithmicScale extends Scale {
 		return ticks;
 	}
 
+	getLabelForValue(value) {
```

Comment:
why do we need this method?

Reply:
Same reason, to display 0 instead of undefined in tooltip.